vegastrike.desktop

```diff
@@ -6,7 +6,7 @@ Comment[cs]=Hraj 3D vesmírnou simulaci
 Comment[es]=Simulador de vuelo espacial 3D/OpenGL
 GenericName[en_US]=vegastrike
 GenericName=vegastrike
-Exec=vs
+Exec=vegastrike
```

Comment:
I'd prefer this to be `vegastrike-engine` with the parameters to the assets. Long term I'd like to get rid of the `vegastrike` binary. This is partly why we had the `vs` wrapper provided.